src/rdb.h

```diff
@@ -40,6 +40,9 @@
  * backward compatible this number gets incremented. */
 #define RDB_VERSION 10
 
+/* Defines related to features that depend on RDB_VERSION. */
+#define RDB_VERSION_STREAM_V2 10
```

Comment:
Touched the top, but I don't about other parsing libs... maybe this should be changed to a new encoding?

Reply:
i guess it'll be the safe thing to do..
we probably won't regret changing the encoding, but there is a chance we'll regret not doing that...

Reply:
Ok, I'll go with a new encoding.

Should I also revert, in this case, the changes I made to carry rdbver into rdbLoad? IIRC this was the only reason to do so.

Reply:
sorry for the late response.
these `rdbver` changes don't cause harm and may be useful in the future, but i suppose it's better to revert rather than introduce something we don't actually need.

Reply:
Sure - resolved via 6ea72d188